main/Main.hs

```diff
@@ -171,13 +176,16 @@ main = do
         . setServerName (toS $ "postgrest/" <> prettyVersion) $
         defaultSettings
 
+  whenLeft socketFileMode $
```

Comment:
```suggestion
  whenLeft socketFileMode panic
```

Reply:
That would allow you to drop the next line: `panic $ (fromJust . leftToMaybe) socketFileMode`.

Reply:
I was not aware of this usage of panic, thanks.